templates/web/oxfordshire/report-video.html

```diff
@@ -0,0 +1,9 @@
+<div class="video-thumbnail-section">
+    <a class="iframe fancybox" rel="fancy" href="https://www.youtube.com/embed/Uf0b_JAzpcY">
```

Comment:
I guess, though I'd rarely suggest this, opening in a new tab might be wanted/expected here, given you are mid reporting flow.

Reply:
@dracos this is one is using a Lightbox(fancybox) to watch the video so we are not leaving the page.